modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/util/test/CalendarUtilTest.java

```diff
@@ -259,6 +267,46 @@ null, createThemeDisplay(), calendarBookings,
 			excpectedCalendarBookingIds, actualCalendarBookingIds);
 	}
 
+	@Test
+	public void testToCalendarJSONObject() throws Exception {
+		CalendarResource calendarResource =
+			CalendarResourceTestUtil.addCalendarResource(
+				_groupLocalService.getGroup(TestPropsValues.getGroupId()));
+
+		_calendarResourceService.updateCalendarResource(
+			calendarResource.getCalendarResourceId(),
+			HashMapBuilder.put(
+				LocaleUtil.getDefault(),
+				"lp'\"></option><img src=x onerror=alert(document.location)>"
```

Comment:
hey @thallescezar please sort the attributes, `onerror` should come before `src`,

after changing it please change the assert as well.

thanks!